packages/react/src/components/Textarea/index.js

```diff
@@ -42,8 +42,11 @@ const Textarea = ({
         {...props}
       />
 
-      {isInvalid && <Icon icon='IconClose' {...defaultIconProps} />}
-      {isValid && <Icon icon='IconCheck' {...defaultIconProps} />}
+      {isInvalid && (
+        <Icon icon={Icon.ICONS.IconAlertOctagon} {...defaultIconProps} />
```

Comment:
Wouldn't it be better to pass down `className` directly? Since all `defaultIconProps` has is that 1 prop.

Reply:
Yep, done 👍